tests/integration/test_vulnerability_detector/conftest.py

```diff
@@ -123,3 +123,10 @@ def clean_cve_tables_func():
     cve_db.clean_all_tables()
     yield
     cve_db.clean_all_tables()
+
+
+@pytest.fixture(scope='function')
+def stop_modules_function():
+    """Stop wazuh modules daemon after finishing a test"""
+    yield
+    control_service('stop')
```

Comment:
It makes no sense for this function to go in the vulnerability detector `conftest`. It is a more generic fixture.

On the other hand, make sure that there is no other fixture that can serve you for this case because I think there are several for this kind of things.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2547/commits/90bed077f7ebf72dd21e5d15b1af1bc674ba3704